Stripe/STPBankAccount.m

```diff
@@ -24,6 +27,33 @@ @implementation STPBankAccount
 
 @synthesize routingNumber, country, currency, accountHolderName, accountHolderType;
 
+#pragma mark - STPBankAccountStatus
+
++ (NSDictionary<NSString *, NSNumber *> *)stringToStatusMapping {
+    return @{
+             @"new": @(STPBankAccountStatusNew),
```

Comment:
Yeah, it is kind of unfortunate that we lose the compiler validating the incomplete switch. Since joey's written thorough tests that include switch statements, I think we'll get the same validation now via tests :)

I think the unknown thing would be an issue whether or not we were using switch statements, since it's a question of what value we return in `...fromString` when the string is invalid. I think the way we're handling this is fine though, just picking an arbitrary fallback unless we really expect to see unknown types from the API.